addons/l10n_cl/views/res_config_settings_view.xml

```diff
@@ -10,11 +9,11 @@
                 <div id="l10n_cl_title" attrs="{'invisible': True}">
                     <h2>Chilean Localization</h2>
                 </div>
-                <div id="l10n_cl_section" class="row mt16 o_settings_container">
+                <div id="l10n_cl_section" class="row mt16 o_settings_container" groups="account.group_account_manager">
                     <!-- inside empty to add configuration of tags -->
+                    <div class="col-12 col-lg-6 o_setting_box" id="l10n_cl_edi_install"/>
```

Comment:
@Danisan I asked you to remove this and you seem not to be using this in l10n_cl_edi, so I wonder if we can merge without this PR (and fix the required after)

Reply:
I will test right now. Thanks!

Reply:
@jco-odoo I have tested and this is unnecessary. Nevertheless, I cannot close or remove it since it has been created by @suv-bmya.